.github/workflows/tidy_up_repository.sh

```diff
@@ -4,26 +4,26 @@ set -o errexit -o nounset -o pipefail
 
 # First verify if the repository is clean.
 if ! git diff --exit-code; then
-	error_msg="ERROR: The repository is not clean - please verify the changes and fix them."
```

Comment:
I don't know why stuff like this changed. I worry it's config files or versions out of sync? :/

Reply:
Maybe we should just stop formatting files in this repo